UnityProject/Assets/Scripts/Lighting/LightSwitchTrigger.cs

```diff
@@ -56,7 +56,7 @@ IEnumerator WaitForLoad()
 
         public override void Interact(GameObject originator, Vector3 position, string hand)
         {
-            if (!PlayerManager.LocalPlayerScript.IsInReach(position, 1.4f))
+            if (!PlayerManager.LocalPlayerScript.IsInReach(position))
```

Comment:
This is meant to prevent through wall switching, please explain if you tested this